app/models/publisher.rb

```diff
@@ -55,6 +55,20 @@ class Publisher < ApplicationRecord
 
   scope :created_recently, -> { where("created_at > :start_date", start_date: 1.week.ago) }
 
+  # publishers that have uphold codes that have been sitting for five minutes
+  # can be cleared if publishers do not create wallet within 5 minute window
+  scope :has_stalled_uphold_code, -> {
```

Comment:
Minor: As these scopes are only used in one place we could consider moving them to the one place only.